TestProject/tests/cellularAutomat/BoardTest.php

```diff
@@ -0,0 +1,176 @@
+<?php
+namespace Tests\CellularAutomat;
+
+use GameOfLife\cellularAutomat\Board;
+use GameOfLife\cellularAutomat\Field;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests whether \CellularAutomat\Board works as expected.
+ */
+class BoardTest extends TestCase
+{
+    /**
+     * @var Board $board Board.
+     * @var int $x Width of the board.
+     * @var int $y Height of the board.
+     */
+    protected $board;
+    protected $x = 5;
+    protected $y = 5;
+
+    /**
+     * Sets height and width of the board.
+     *
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        $this->board = new Board(10, 10);
+    }
+
+    /**
+     * Checks whether the board - a two-dimensional array - is empty.
+     *
+     * @param Field[][] $_board
+     * @return bool Is Empty or not.
+     */
+    private function checkBoardArrayEmpty(array $_board): bool
+    {
+        $boardIsEmpty = true;
+        foreach ($_board as $row)
+        {
+            foreach ($row as $cell)
+            {
+                if ($cell != 0)
+                    $boardIsEmpty = false;
+            }
+        }
+        return $boardIsEmpty;
+    }
+
+    /**
+     * Asserts that width has the same value that was defined inside the constructor.
+     *
+     * @return void
+     */
+    public function testChecksConstructorWidthIsActuallyWidth()
+    {
+        $this->assertEquals(10, $this->board->getWidth());
+    }
+
+    /**
+     * Asserts that height has the same value that was defined inside the constructor.
+     *
+     * @return void
+     */
+    public function testChecksIfConstructorHeightIsActuallyHeight()
+    {
+        $this->assertEquals(10, $this->board->getHeight());
+    }
+
+    /**
+     * Asserts that the created board  is empty.
+     */
+    public function testBoardIsEmpty()
+    {
+        $board = $this->board->getFieldBoard();
+        $boardIsEmpty = $this->checkBoardArrayEmpty($board);
+
+        $this->assertTrue($boardIsEmpty);
```

Comment:
Please combine this into one line, no need to define a variable and only use it once.